pages/windows/wget.md

```diff
@@ -0,0 +1,15 @@
+# wget
```

Comment:
Same here. Let's remove these `wget` and `curl` pages from this pull request.

Reply:
I wrote these two pages for two reasons. First is the fact that in PowerShell for Windows, both commands will be substituted with `Invoke-WebRequest` with incompatible set of accepted parameters when `wget` or `curl` is not visible on `%PATH%`:

![image](https://user-images.githubusercontent.com/17312341/177046493-0fb4ead2-742c-4036-aca3-7814c636bc72.png)

and similarly, executing `curl` and `wget` without additional subcommand or parameters will still return the same prompt with `Invoke-WebRequest` if both aren't installed on `%PATH%`:

![image](https://user-images.githubusercontent.com/17312341/177046636-abdb9b91-1f35-4718-b5a9-22654b523521.png)

As far as I know there are some edge cases where a command documentation is split between `common` and OS-specific versions, e.g. `common/cd` and `windows/cd`. As long as `tldr` clients still can show the common or platform-specific versions of the same command I still prefer to keep `windows/curl` and `windows/wget` this way to avoid confusion among those PowerShell users.

Ah I forgot that the original documentation should be access with `tldr {{curl|wget}} --platform common` instead of `tldr common/{{curl|wget}}`.

Reply:
Oh I see, thanks for clarifying this. This is a rather confusing situation, isn't it?

 - `cmd.exe` is the main shell for Windows, which if you have `curl` or `wget` installed works as on Linux
 - PowerShell is also an official shell distributed and enabled by default on Windows, where `curl` and `wget` resolve → `Invoke-WebRequest`

With this in mind, your solution here seems to be the best option. Thanks for explaining - not being a Windows user I didn't understand at first.